src/tiles/collection-browser-loading-tile.ts

```diff
@@ -0,0 +1,29 @@
+import { css, html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+@customElement('collection-browser-loading-tile')
+export class CollectionBrowserLoadingTile extends LitElement {
+  render() {
+    return html` <div id="container"></div> `;
```

Comment:
QUESTION: Why the space around the tag? Why force whitespace around the tag?

Reply:
This looks like `prettier`'s auto-formatting. I'm not sure why it does that or which setting causes it, but I've seen it quite a bit.